lib/ruby_lsp/requests/folding_ranges.rb

```diff
@@ -202,6 +203,11 @@ def on_call_node_enter(node)
         add_lines_range(location.start_line, location.end_line - 1)
       end
 
+      sig { params(node: Prism::LambdaNode).void }
+      def on_lambda_node_enter(node)
+        add_simple_range(node)
```

Comment:
Is this sufficient? Do we need to also handle multiline parameters here?

Reply:
It depends on the behaviour you want for how something like:
```ruby
foo = -> (
  bar,
  baz
) do
  # something
end
```
to fold.

Should it be:
```ruby
foo = -> (
  ...
end
```
or
```ruby
foo = -> (
  bar,
  baz
) do
  ...
end
```
If you want the latter behaviour, you need to do handling similar to the one `on_def_node_enter` does, and add a test case to ensure that behaviour.

Reply:
For now I'll stick with this basic behavior and if anyone asks for the folding of multiline arguments in lambdas I'll be happy to implement it.

I've also corrected the rubocop offense. Thank you for the reply.

Reply:
Yeah, for multiline parameters, we'll need something similar to what we have implemented in `on_def`.

But we can definitely move forward with this and address the parameters in the next PR.